README.md

```diff
@@ -25,7 +25,40 @@ kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/
 
 <!---x-release-please-end-->
 
-### Architecture
+### Release contents
+
+- `certificate.yaml` holds the cert-manager manifests used to authorize requests between components
+- `release.yaml` contains the configuration of
+  - `FeatureFlagConfiguration` `CustomResourceDefinition` (custom type that holds the configured state of feature flags)
+  - standard kubernetes primitives (e.g. namespace, accounts, roles, bindings, configmaps)
+  - operator controller manager service
+  - operator webhook service
+  - deployment with containers kube-rbac-proxy & manager
+  - `MutatingWebhookConfiguration` (configures webhooks to call the webhook service)
+
+### How to deploy a flag consuming application
+
+_Prerequisite: the release and certificates have been deployed as outlined above._
+
+Deploying a flag consuming application requires (at minimum) the creation of the following 2 resources (an example can be found [here](./config/samples/end-to-end.yaml)):
```

Comment:
It's not true to say you have to use the resource type of "Deployment" it could also be an STS/DS or orphaned RS. Maybe to say in this piece that it should be one of those

Reply:
Good point, thank you. I've included STS & DS alongside Deployment in the heading